rocket_twin/tuto_marc_val/ball_to_space.py

```diff
@@ -0,0 +1,50 @@
+from cosapp.base import System
+from cosapp.drivers import RungeKutta
+from cosapp.recorders import DataFrameRecorder
+import numpy as np
+import matplotlib.pyplot as plt
+
+class Ball(System):
+
```

Comment:
why did you change this file ?

Reply:
Just because we needed the numpy library to generate array I think